visualize.Rmd

```diff
@@ -1,115 +1,114 @@
-# Data visualisation
+# Visualización de datos
 
-## Introduction
+## Introducción
 
-> "The simple graph has brought more information to the data analyst’s mind 
-> than any other device." --- John Tukey
+>"Un simple gráfico ha brindado más información a la mente del analista de datos
+>que cualquier otro dispositivo". --- John Tukey
 
-This chapter will teach you how to visualise your data using ggplot2. R has several systems for making graphs, but ggplot2 is one of the most elegant and most versatile. ggplot2 implements the __grammar of graphics__, a coherent system for describing and building graphs. With ggplot2, you can do more faster by learning one system and applying it in many places.
+En este capítulo aprenderás cómo visualizar datos usando el paquete **ggplot2**. De los muchos sistemas que posee R para hacer gráficos, **ggplot2** es uno de los más elegantes y versátiles. Esto se debe a que implementa  un sistema coherente para describir y construir gráficos, conocido como la __gramática de gráficos__. Con **ggplot2** puedes hacer más cosas en menor tiempo, aprendiendo un único sistema y aplicándolo en muchos lugares
 
-If you'd like to learn more about the theoretical underpinnings of ggplot2 before you start, I'd recommend reading "The Layered Grammar of Graphics", <http://vita.had.co.nz/papers/layered-grammar.pdf>.
+Si deseas obtener más información sobre los fundamentos teóricos de **ggplot2** antes de comenzar, te recomendamos leer "La gramática de gráficos en capas", <http://vita.had.co.nz/papers/layered-grammar.pdf>.
```

Comment:
, te recomendamos leer "La gramática de gráficos en capas", <http://vita.had.co.nz/papers/layered-grammar.pdf>

, te recomendamos leer "La gramática de gráficos en capas (*The Layered Grammar of Graphics*) ", <http://vita.had.co.nz/papers/layered-grammar.pdf>

Se cita un trabajo que está originalmente escrito en Inglés.

Reply:
De acuerdo con esta revisión. El párrafo quedaría entonces: 

> Si deseas obtener más información sobre los fundamentos teóricos de **ggplot2** antes de comenzar, te recomendamos leer "La gramática de gráficos en capas (*The Layered Grammar of Graphics*)", <http://vita.had.co.nz/papers/layered-grammar.pdf>.